modules/plugin/mongodb/src/test/java/org/geotools/data/mongodb/FilterToMongoTest.java

```diff
@@ -210,6 +211,29 @@ public void testWithin() {
                 "urn:x-mongodb:crs:strictwinding:EPSG:4326", filterIntersectsCrsPropertiesName);
     }
 
+    @Test
+    public void testDWithin() {
```

Comment:
Since you're dealing with logic filters as well, we'll need test to cover that too.

Reply:
No needed anymore (since we don't change this logic)